config/envs/default.py

```diff
@@ -154,6 +158,18 @@ class DefaultConfig(object):
         FUND_STORE_API_HOST + CommonConfig.ROUND_ENDPOINT
     )
 
+    # Post-award frontend
+    POST_AWARD_FRONTEND_HOST = environ.get("POST_AWARD_FRONTEND_HOST", "")
+    POST_AWARD_FRONTEND_LOGIN_URL = POST_AWARD_FRONTEND_HOST + "/"
+
+    # Safe list of return applications
+    SAFE_RETURN_APPS = {
+        "post-award-frontend": SafeAppConfig(  # TODO: Change to fsd_utils.auth.config.SupportedApp.POST_AWARD_FRONTEND
```

Comment:
Any reason we can't do this TODO just now?

Are you waiting for https://github.com/communitiesuk/funding-service-design-utils/pull/78 to be merged?

Reply:
This has just been merged as I submitted this review

Reply:
Yeah just updated this 👍